gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/InstrumentatedGoogleHadoopFileSystem.java

```diff
@@ -70,7 +72,14 @@ protected void configureBuckets(GoogleCloudStorageFileSystem gcsFs) throws IOExc
   @Override
   public FSDataInputStream open(Path hadoopPath, int bufferSize) throws IOException {
     entryPoint(GhfsStatistic.INVOCATION_OPEN);
-    return super.open(hadoopPath, bufferSize);
+    FSDataInputStream resp;
+    try {
+      resp = super.open(hadoopPath, bufferSize);
+    } catch (IOException e) {
+      throw e;
+    }
```

Comment:
This is a no-op, do we need this change ?

Reply:
Updated